src/coreclr/jit/hwintrinsiclistarm64sve.h

```diff
@@ -312,8 +312,8 @@ HARDWARE_INTRINSIC(Sve,           ZipLow,
 #define SPECIAL_NI_Sve            NI_Sve_ConditionalExtractAfterLastActiveElementScalar
 HARDWARE_INTRINSIC(Sve,           ConditionalExtractAfterLastActiveElementScalar,                    0,      3,     {INS_sve_clasta,     INS_sve_clasta,     INS_sve_clasta,     INS_sve_clasta,     INS_sve_clasta,     INS_sve_clasta,     INS_sve_clasta,     INS_sve_clasta,     INS_sve_clasta,     INS_sve_clasta},  HW_Category_Scalar,                HW_Flag_Scalable|HW_Flag_SpecialCodeGen|HW_Flag_SpecialImport|HW_Flag_HasRMWSemantics|HW_Flag_ExplicitMaskedOperation|HW_Flag_LowMaskedOperation)
 HARDWARE_INTRINSIC(Sve,           ConditionalExtractLastActiveElementScalar,                         0,      3,     {INS_sve_clastb,     INS_sve_clastb,     INS_sve_clastb,     INS_sve_clastb,     INS_sve_clastb,     INS_sve_clastb,     INS_sve_clastb,     INS_sve_clastb,     INS_sve_clastb,     INS_sve_clastb},  HW_Category_Scalar,                HW_Flag_Scalable|HW_Flag_SpecialCodeGen|HW_Flag_SpecialImport|HW_Flag_HasRMWSemantics|HW_Flag_ExplicitMaskedOperation|HW_Flag_LowMaskedOperation)
-HARDWARE_INTRINSIC(Sve,           ConvertMaskToVector,                                              -1,      1,     {INS_sve_mov,        INS_sve_mov,        INS_sve_mov,        INS_sve_mov,        INS_sve_mov,        INS_sve_mov,        INS_sve_mov,        INS_sve_mov,        INS_sve_mov,        INS_sve_mov},     HW_Category_Helper,                HW_Flag_Scalable|HW_Flag_ExplicitMaskedOperation)
-HARDWARE_INTRINSIC(Sve,           ConvertVectorToMask,                                              -1,      2,     {INS_sve_cmpne,      INS_sve_cmpne,      INS_sve_cmpne,      INS_sve_cmpne,      INS_sve_cmpne,      INS_sve_cmpne,      INS_sve_cmpne,      INS_sve_cmpne,      INS_sve_cmpne,      INS_sve_cmpne},   HW_Category_Helper,                HW_Flag_Scalable|HW_Flag_ReturnsPerElementMask|HW_Flag_LowMaskedOperation)
+HARDWARE_INTRINSIC(Sve,           ConvertMaskToVector,                                              -1,      1,     {INS_sve_mov,        INS_sve_mov,        INS_sve_mov,        INS_sve_mov,        INS_sve_mov,        INS_sve_mov,        INS_sve_mov,        INS_sve_mov,        INS_sve_mov,        INS_sve_mov},     HW_Category_Helper,                HW_Flag_Scalable)
+HARDWARE_INTRINSIC(Sve,           ConvertVectorToMask,                                              -1,      2,     {INS_sve_cmpne,      INS_sve_cmpne,      INS_sve_cmpne,      INS_sve_cmpne,      INS_sve_cmpne,      INS_sve_cmpne,      INS_sve_cmpne,      INS_sve_cmpne,      INS_sve_cmpne,      INS_sve_cmpne},   HW_Category_Helper,                HW_Flag_Scalable|HW_Flag_ExplicitMaskedOperation|HW_Flag_ReturnsPerElementMask|HW_Flag_LowMaskedOperation)
```

Comment:
removing `HW_Flag_ExplicitMaskedOperation` from `MaskToVector` and putting it `VectorToMask` makes sense, but wondering why it worked so far? probably we do not care of this flag and can be eliminated?

Reply:
spoke offline. As expected, this was never used for these 2 intrinsics, but good to have for correctness.

Reply:
We'll never use it because the converts will never be embedded